pages/common/gpg-keyserver.md

```diff
@@ -0,0 +1,17 @@
+# gpg --keyserver
+
+> GNU Privacy Guard, work with keyservers
+> See `gpg2` for GNU Privacy Guard 2.
+> More information: <https://gnupg.org>.
+
+- Encrypt a directory into `archive.gpg` using a passphrase:
+
+`gpg --keyserver pool.sks-keyservers.net --recv-key {{fingerprint}}`
+
+- Decrypt `archive.gpg` into a directory of the same name:
+
+`gpg-zip --decrypt {{path/to/archive.gpg}}`
```

Comment:
Please use another branch *don't close this PR yet*, but `gpg-zip` should be a page with just `gpg-zip` examples, not merged with `gpg --keyserver` that should be in another PR too.

Reply:
Sorry that change should not be in this pr 🤦